### PR TITLE
import csv layout

### PR DESCRIPTION
**Pull request by ghost**
## Description
Adding button import file csv
## Reference Requirements
- Product Backlog ID: #PBI-16
- Task ID: #16.2

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/backend/books/list.blade.php [hunk 1]
```diff
@@ -49,11 +50,26 @@
...
+                                          </li>
+                                          <li>
+                                            <form id="import-form" action="" method="post" enctype="multipart/form-data">
+                                              {{csrf_field()}}
+                                              <input type="file" name="import-data" class="form-control" id="import-book">
+                                            </form>
+                                          </li>
+                                        </ul>
+                                      </li>
+                                    </ul>
```
Comment (at-annguyen) [inline resources/views/backend/books/list.blade.php]: what is this?

File: public/js/excel.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    $('#confirmImport').modal('show');
+  })
+  //reset file input
+  $('#cancel-import').click(function() {
+    $('#import-book').val('');
+  })
+  $('#confirm-import').click(function() {
+    $('#import-form').submit();
+  })
+})
```
Comment (at-annguyen) [inline public/js/excel.js]: Needed a new line

File: public/js/excel.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        $next = $this.next();
+
+    $next.slideToggle();
+    $this.parent().toggleClass('open');
+
+    if (!e.data.multiple) {
+      $el.find('.submenu').not($next).slideUp().parent().removeClass('open');
+    };
+  }
+
```
Comment (at-annguyen) [inline public/js/excel.js]: what is this?

File: resources/views/backend/books/list.blade.php [hunk 4]
```diff
@@ -49,11 +50,26 @@
...
                         <!-- add form search and select for book -->
                         <!-- start -->
-                        <form action="{{ route('books.index') }}" method="GET" id="frm-search">
                             <div class="form-row">
-                                <div class="form-group col-md-3">
-                                    <a class="btn btn-success" href="{{ route('books.create') }}">{{ __('books.add_book') }}</a>
+                                <div class="col-md-3">
+                                    <ul id="accordion" class="accordion">
+                                      <li>
+                                        <div class="link"><i class="fa fa-book"></i>NeverMore<i class="fa fa-chevron-down"></i></div>
```
Comment (at-annguyen) [inline resources/views/backend/books/list.blade.php]: multi language???

**Comment by at-annguyen** [comment]
LGTM

**Comment by at-vietpham** [comment]
LGTM 🏯

**Pull request merged**